src/FlightDisplay/FlyView.qml

```diff
@@ -27,13 +27,23 @@ import QGroundControl.Palette
 import QGroundControl.ScreenTools
 import QGroundControl.Vehicle
 
+
+// 3D Viewer modules
+import Viewer3DQmlType             1.0
```

Comment:
The new Qt6 standard is to no include module version numbers.

Reply:
Right, I will remove the versions.